modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/fragment/renderer/FDSAdminFragmentRenderer.java

```diff
@@ -304,6 +304,8 @@ private String _buildFragmentHTML(
 				"namespace", fragmentRendererContext.getFragmentElementId()
 			).put(
 				"pagination", _getPaginationJSONObject(fdsViewObjectEntry)
+			).put(
+				"parameters", fdsViewObjectEntry.getPropertyValue("parameters")
```

Comment:
Overall approach looks good to me. I'm a bit concerned with the naming, in particular, `parameter` looks to broad. To begin, the endpoint URL may already contain something we could misinterpret as parameter in the path, e.g. `/v1.0/user-accounts/{userAccountId}/organizations`, which are totally unrelated to the feature we're introducing here.

Also, there is the idea that those parameters are not the only ones that will end up being added to the final URL used by FDS to fetch data. Pagination, filter, nested fields are dynamically added by the FDS/Fragment when needed.

Having said that, I'd like to start a short discussion about what should be the ideal name for this.

Considering we're storing it as string, a reasonable name to me would contain mentions to "query", "string" and maybe "endpoint", to make it more accurate. It might be useful to highlight these ones are _additional_ to signal they're not exclusive (implementation will include them but others too)

Something along the lines of `enpdointQueryString`, `queryString`, `additionalQueryString` might work here.

Please note I'm talking from the DSM/FDS api perspective. UI can keep the name product/design feels best for the end user intended purpose. @thektan @herrardo

Reply:
Ah, yes I agree with that. Those suggestions all sound good. 

I suppose a strange thing with this `parameters` is it feels like it should be part of the `apiURL`, but it needs to be passed in separately so it can be merged with the other configured `sorts`/`filters`.

So I'm thinking `additionalQueryString` so it's clear it's more of like an "add on".

Reply:
I know that `parameters` is quite broad but is still accurate: https://support.google.com/google-ads/answer/6277564?hl=en not that they use 'URLParameters'

It's true that there are another names such as `queryString` or even the `URLSearchParams` interface used in JS.

Also, we know they will other parameters (pagination, filtering and sorting coming from the DSM) but they are the same kind of parameters so i would name them in the same fashion, adding a prefix to the ones treated in this PR, so, possibilities:

- URLParameters  (the ones from DSM)
- URLExtraParameters (the ones in this PR)

- URLParameters  (the ones from DSM)
- URLStaticParameters (the ones in this PR)

- QueryString(the ones from DSM)
- AdditionalQueryString (the ones in this PR)

Just ideas to see is you like them ;)

Reply:
Ended up going with `queryString` to keep it simple for a field name on the `FDSView` object as well.

Reply:
So Dani pointed out that I should add the new field from this PR to [migration to JSON](https://github.com/liferay-frontend/liferay-portal/pull/4414). Without context of discussion here, I found `queryString` to be very confusing. We typically associate `query` with search, and `*String` seems unnecessary.

I'd call this `additionalURLParameters`.

Reply:
This is good with me. I'll rename this to `additionalAPIURLParameters` in the meantime.

`queryString` made sense to me since it seems to be an accurate description of the value. For example, a wiki on [Query String](https://en.wikipedia.org/wiki/Query_string). There's also a popular js package called [`query-string`](https://github.com/sindresorhus/query-string) which interprets strings like `?foo=bar`.

That being said, `additionalAPIURLParameters` does seem appropriate in the context of DSM and FDS because of the naming used in the UI and how it's being combined with the `apiURL`.